corehq/apps/app_manager/tests/data/suite/search_command_detail.xml

```diff
@@ -95,12 +95,12 @@
     <field>
       <header>
         <text>
-          <locale id="m0.case_long.case_whatever_2.header"/>
+          <locale id="m0.search_short.case_whatever_2.header"/>
         </text>
       </header>
       <template>
         <text>
-          <xpath function="instance('ledgerdb')/ledgers/name/name"/>
+          <xpath function="instance('reports')/report[1]/name"/>
```

Comment:
I'm idly curious why this changed in this PR...?

Reply:
the diff for this commit is weird. If you check the diff of the whole range you'll see that I added `case_long` and `search_short` sections to the detail.